src/main/java/com/tekenable/config/AuditConfigurator.java

```diff
@@ -10,26 +14,24 @@
 
```

Comment:
Can we turn off Auto-commit on the JDBC template and call commit at the very end? That way if there is an exception returned from any stage of the Audit creation that we rollback everything instead of getting half way through the update and then failing?